### ISSUE TITLE
Why malware software?

### ISSUE THREAD

**Issue by zhangyu2020** [ShareX/ShareX#5080]
One day I wanted to install this software on my company's laptop(win10 64). when I double-clicked the .exe file to start installing, the dialog box popup-up appeared saying 'this is a malware, the software installation was blocked' some words like that, I was not able to install anymore, then I went to 'Control Panel > Security and Maintenance > Virus & threat protection', I found the providers for protecting the computer became 'CrowdStrike Falcon Sensor' since the company implemented a security update,  I am not sure if some code violated the policy of .net framework or the specific MS security measures.  pls verifying this issue, Thanks a lot!
ShareX version:  13.2.1 (downloaded from the official website)
Windows 10 Enterprise, 64-bit  version: 1909

**Comment by L1Q** [comment]
You have antivirus software installed on your company's laptop, looks like the software is "CrowdStrike Falcon".
Provided you have downloaded ShareX from a reputable source, this is a false positive.
Valid download sources are: [github releases](https://github.com/ShareX/ShareX/releases), [getsharex.com](https://getsharex.com/) (also points to github), [Steam](http://store.steampowered.com/app/400040/), or [MS store](https://www.microsoft.com/store/apps/9nblggh4z1sp)
Basically blame CrowdStrike and/or your company for this.
Meanwhile try installing from [Microsoft Store](https://www.microsoft.com/store/apps/9nblggh4z1sp).

**Comment by zhangyu2020** [comment]
Yes, I downloaded it from https://getsharex.com/ for sure
